applications/PfemFluidDynamicsApplication/python_scripts/MainFluidPFEM.py

```diff
@@ -384,4 +385,5 @@ def _import_project_parameters(self, input_file):
 
 
 if __name__ == "__main__":
-    Solution().Run()
+    model = Model()
```

Comment:
I am not in the branch and I can't test it, but it looks like `Model()` wouldn't be recognized here, right?
```suggestion
    model = KratosMultiphysics.Model()
```

Reply:
Thanks Miguel Ángel. The branch was not compiling so I could not verify if our changes were working.